plugins/commands/requirements/src/main/kotlin/RequirementsCommand.kt

```diff
@@ -141,8 +148,27 @@ class RequirementsCommand : OrtCommand(
         return statusCode
     }
 
-    private fun getToolsByCategory(): Map<String, List<CommandLineTool>> {
-        val reflections = Reflections("org.ossreviewtoolkit", Scanners.SubTypes)
+    private fun listPlugins(reflections: Reflections) {
+        val pluginClasses = reflections.getSubTypesOf(Plugin::class.java)
+
+        val pluginTypes = pluginClasses.mapNotNull { clazz ->
+            val companion = clazz.declaredClasses.find { it.name.endsWith("\$Companion") }
+            val all = runCatching { companion?.getDeclaredMethod("getALL") }.getOrNull()
```

Comment:
As this line is relatively easy to break, shall we add a test which verifies the at least one plugin is listed?

Also is "getALL" actually right, or should it be "getAll"?

Reply:
> Also is "getALL" actually right, or should it be "getAll"?

It's correct, as it's Java syntax to refer to Kotlin's `ALL` property.